client/tests/end2end/test-globaleaks-process.js

```diff
@@ -277,6 +277,20 @@ describe('globaLeaks process', function() {
     });
   });
 
+  it('Recipient should be able to export the first submission from the tips page', function(done) {
+    login_receiver(receiver_username, receiver_password).then(function() {
+      element(by.css('#tip-0 form.tipExport button')).click().then(function() {
+        if (utils.testFileDownload()) {
```

Comment:
the protractor selector (that inherits from webdriver) allow also nested lookups like:
```element(by.id('tip-0')).element(by.id('tip-action-export')).click().then(function() {```